opentelemetry-instrumentation/src/opentelemetry/instrumentation/utils.py

```diff
@@ -115,3 +116,38 @@ def _start_internal_or_server_span(
         attributes=attributes,
     )
     return span, token
+
+
+_KEY_VALUE_DELIMITER = ","
+
+
+def _generate_sql_comment(**meta):
```

Comment:
Is it really necessary to add this to `utils`? This is a single-use function, its contents can just be used directly where the function is called.

Reply:
I think so, there will be many sql db instrumentations that will rely on this.